### PR TITLE
Install chipsec from endpoints UI

### PR DESCRIPTION
**Pull request by maziland**
Added the install chipsec action on endpoints.
Also added the use of Toast messages on the UI, for a nice user experience

### The action menu
Screenshot 2024-07-22 at 9 14 07 AM

### After choosing an action:
Screenshot 2024-07-22 at 9 14 23 AM
Screenshot 2024-07-22 at 9 14 57 AM

### CODE REVIEW

File: scripts/install_chipsec.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 set -e
 lsmod | grep chipsec 1>/dev/null 2>&1 && echo "Chipsec already insalled :)" && exit
-python3 -m pip install -U pip && python3 -m pip install -U setuptools
+(python3 -m pip install -U pip && python3 -m pip install -U setuptools) || true
```
Comment (GeneralSnuggles) [inline scripts/install_chipsec.sh:4]: What does this do?

**Review by GeneralSnuggles — APPROVED** [review]
Looks good
Need to make sure public/js/endpoint.js and the ejs aren't destroyed when merging

File: scripts/install_chipsec.sh (hunk 1 above)
Comment (maziland) [reply scripts/install_chipsec.sh:4]: Some python environments does not allow installing setup tools, so if it fails I return true from this line.

**Review by tal-az — APPROVED** [review]
Looks superb

File: controllers/chipsec.js [hunk 2]
```diff
@@ -36,13 +36,31 @@ async function installChipsec(req, res) {
 
 async function uninstallChipsec(req, res) {
     try {
-        //run fleet api to run fleet script to uninstall chipsec
-        // response = await fleetService.fleetApiGetRequest(/**enter here api to run scripts */)
-        res.status(200).send("Chipsec Uninstalled Successfuly!")
+        scripts = await fleetService.listScripts();
+        uninstallChipsecScriptId = scripts.find(script => script.name === "uninstall_chipsec.sh").id
+
+        hostId = req.params.hostId
```
Comment (tal-az) [inline controllers/chipsec.js:41]: validate parameter

**Review by tal-az — COMMENTED** [review]
Looks good, commented a minor validation check

**Pull request merged**